tests/unit_benchmarks/benchmarks.sh

```
@@ -0,0 +1,179 @@
+#!/bin/bash
+
+PROGNAME="${BASH_SOURCE[0]}"
+HERE="$(cd "$(dirname "$PROGNAME")" &>/dev/null && pwd)"
+ROOT=$(cd $HERE/../.. && pwd)
+READIES=$ROOT/deps/readies
+. $READIES/shibumi/defs
+
+cd $HERE
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)